index.js

```diff
@@ -3,17 +3,17 @@ const path = require('path');
 const got = require('got');
 const pMap = require('p-map');
 const streamz = require('streamz');
-const unzipper = require('unzipper').Parse;
+const unzipper = require('unzip-stream').Parse;
```

Comment:
Ca change quoi `unzipper` est deprecie ?

Reply:
When I use "unzipper" I have an error "Error: Cannot find module 'unzip' but in "unzip-stream" I have access to this method

Reply:
Ca doit être lié a ta version de node.js je pense faudrait vérifié plus en détails.